Gemfile.lock

```diff
@@ -927,7 +939,7 @@ GEM
       rubyzip (~> 2.0.0)
 
 PLATFORMS
-  ruby
```

Comment:
Platform has changed and needs locking for Ruby

Reply:
is there a way to do this without hacking Gemfile.lock?

Reply:
I had to do
```
 bundle lock --add-platform ruby
 bundle lock --remove-platform x86_64-linux
```
the remove-platform might not be necesary for you